docker/pulp/entrypoint.sh

```diff
@@ -62,7 +62,7 @@ run_resource_manager() {
   _prepare_env
   exec rq worker \
       -w 'pulpcore.tasking.worker.PulpWorker' \
-      -n 'resource-manager@%h' \
+      -n 'resource-manager@automation-hub' \
```

Comment:
While we are changing this, might as well use the long args (--name) here to make this more self documenting.

Reply:
Looking at rq docs https://python-rq.org/docs/workers/#worker-names, the docs seem to suggest not naming the workers and letting it create the default random names. Do we need to name the workers or could we avoid it?